.test-infra/jenkins/common_job_properties.groovy

```diff
@@ -260,14 +260,19 @@ class common_job_properties {
     context.steps {
         // Clean up environment.
         shell('rm -rf PerfKitBenchmarker')
+        shell('rm -rf .env')
```

Comment:
Can this deletion break other tests running in the same VM ?

Reply:
No - those paths are relative to build's workspace, so it's independent from other builds running on the same worker - it just cleans environment after last build.